### PR TITLE
Start 1.16.0 dev

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
**What this PR does / why we need it**:

Post `v1.15.0` release. Tagging master with `v1.16.0-dev`

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.15.0-dev
+1.16.0-dev
```
Comment (cliveseldon) [inline version.txt:1]: I assume the file/examples in this PR will be usable only after images are built which would be 30mins after merge or so?

File: doc/source/reference/upgrading.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ The label on services pointing to containers for each node in your inference gra
 ### OpenShift default storage initializer
 
 In this patch release we fixed the default storage initializer used on OpenShift (both Community and Certified operators).
-The image used now is `seldonio/rclone-storage-initializer:1.14.1` which is the same as one used for the non-OpenShift releases (*).
+The image used now is `seldonio/rclone-storage-initializer:1.16.0-dev` which is the same as one used for the non-OpenShift releases (*).
```
Comment (RafalSkolasinski) [inline doc/source/reference/upgrading.md:24]:
```suggestion
The image used now is `seldonio/rclone-storage-initializer:1.14.1` which is the same as one used for the non-OpenShift releases (*).
```

[the remark this answers is not in the document]
Comment (RafalSkolasinski) [reply doc/source/reference/upgrading.md]: Good spot, I fixed it. Maybe we will need to look why upgrading script touches this file and make it not do it.

File: version.txt (hunk 1 above)
Comment (RafalSkolasinski) [reply version.txt:1]: Yes, that is the case after beginning of every new minor version dev.

**Pull request merged**